src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -15,7 +15,7 @@
       "default": false,
       "description": "Additionally install NVIDIA Tools Extension (NVTX)"
     },
-    "cudaVersion": {
```

Comment:
I know this is still a draft PR. However, do we really want to rename the option? Renaming options seems breaking/unexpected change and we don't have back compat setup. Isn't description sufficient to explain the same?

However, if we still want to rename the option, then we'd like to bump the major version.